storage/api/Storage/ListFiles.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Storage.V1;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Storage
+{
+    public class ListFiles
+    {
+        // [START storage_list_files]
+        public static List<Google.Apis.Storage.v1.Data.Object> GetFileList(string bucketName)
+        {
+            var storage = StorageClient.Create();
+            var storageObjects = storage.ListObjects(bucketName, "").ToList();
+            foreach (var storageObject in storageObjects)
+            {
+                Console.WriteLine(storageObject.Name);
+            }
+
+            return storageObjects;
+        }
+        // [END storage_list_files]
+
+        // [START storage_list_files_with_prefix]
```

Comment:
This should be a separate class, and should have documentation explaining how listing with delimiters works. See the other language versions of this sample for examples on what that documentation can look like. i.e. https://cloud.google.com/storage/docs/listing-objects#storage-list-objects-go

Reply:
Done!!